chain/test/Test/Pos/Chain/Txp/Arbitrary.hs

```diff
@@ -74,7 +74,7 @@ genTxInWitness pm = oneof
     [ PkWitness <$> arbitrary <*> genSignature pm arbitrary
       -- this can generate a redeemer script where a validator script is
       -- needed and vice-versa, but it doesn't matter
-    , ScriptWitness <$> arbitrary <*> arbitrary
+
```

Comment:
We should probably remove this whitespace and the comment above

Reply:
I alrady had that as a local clean up that I forgot to push :).